lib/go/contracts/contracts.go

```diff
@@ -361,3 +367,11 @@ func TestFlowFees(fungibleTokenAddress, flowTokenAddress, storageFeesAddress str
 
 	return []byte(code)
 }
+
+func ExampleToken(env templates.Environment) []byte {
+	return ftcontracts.ExampleToken(env.FungibleTokenAddress, env.MetadataViewsAddress, env.FungibleTokenMetadataViewsAddress)
+}
+
+func ExampleNFT(env templates.Environment) []byte {
+	return nftcontracts.ExampleNFTWithCrossVMPointers(flow.HexToAddress(env.NonFungibleTokenAddress), flow.HexToAddress(env.MetadataViewsAddress), flow.HexToAddress(env.ViewResolverAddress), flow.HexToAddress(env.EVMAddress), flow.HexToAddress(env.CrossVMMetadataViewsAddress))
+}
```

Comment:
I'm assuming these are just added for testing?

Reply:
yeah, I added them so we could use them for testing in flow-go